packages/client-python/src/rocketride/cli/commands/runtime.py

```
@@ -0,0 +1,888 @@
+    return f'{sys.executable} -m rocketride'
+
+
+def _is_autostart_enabled() -> bool:
+    """Check whether the OS autostart hook is currently registered."""
+    if sys.platform == 'win32':
+        import winreg
+
+        try:
+            with winreg.OpenKey(winreg.HKEY_CURRENT_USER, _autostart_registry_key()) as key:
+                winreg.QueryValueEx(key, _autostart_registry_value_name())
+            return True
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)